recipes/intelligent_queue_management/README.md

```diff
@@ -104,4 +104,33 @@ _NOTE: Alternatively, you can run all steps with the following command:_
 python main.py --stream sample_video.mp4
 ```
 
-Congratulations! You have successfully set up and run the Intelligent Queue Management application with OpenVINO™.
+## Step 4
+
+Benchmark the Model with OpenVINO's Benchmark_App
+Benchmarking provides insight into your YOLOv8 model's real-world performance. Performance may vary based on use and configuration.
+
+### Benchmark Results 
+
+![YOLOv8m Benchmark Results](https://github.com/AnishaUdayakumar/intelligent-queue-management-openvino/assets/109281183/8a81243e-ee32-4b30-9994-326ecb07d32f)
+
+The benchmarks for the YOLOv8m model were run using OpenVINO version 2023.0.0 on the 4th Gen Intel® Xeon® Scalable Processor:
```

Comment:
The configuration is missing for benchmarking and this is what is required by the legal team.

Reply:
I have updated the statement with configuration details. Please let me know if any further changes are needed.